README.md

```diff
@@ -469,6 +469,8 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [React Patterns](https://reactpatterns.com/)
 - [Useful Packages you should know in React](https://medium.com/better-programming/14-useful-packages-every-react-developer-should-know-55b47a325d3)
 - [Learn React Test Driven Development](https://learntdd.in/react/)
+- [ReactJS Tutorial for Begiiners] (https://youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3)
```

Comment:
Please fix the title as it has a typo. Also, remove the space between ] and ( as the link is now invalid.

Reply:
Sure will fix the error